src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -193,15 +205,16 @@ public static Dictionary<string, OrganelleEfficiency> ComputeOrganelleProcessEff
     ///   Computes the energy balance for the given organelles in biome and at a given time during the day (or type
     ///   can be specified to be a different type of value)
     /// </summary>
-    public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
+    public static void ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
         IBiomeConditions biome, MembraneType membrane, bool includeMovementCost, bool isPlayerSpecies,
-        WorldGenerationSettings worldSettings, CompoundAmountType amountType, bool calculateRequiredResources)
+        WorldGenerationSettings worldSettings, CompoundAmountType amountType,
+        [NotNull] ref EnergyBalanceInfo? result)
```

Comment:
If this is constrained to be not null, what is the point with this being marked nullable and a ref? I really think the simplest thing is to always just require the called to already have created the instance...